src/k8s/api/v1/types.go

```diff
@@ -33,6 +33,27 @@ func (b *BootstrapConfig) ToMap() (map[string]string, error) {
 	}, nil
 }
 
+func (b *BootstrapConfig) IsValidCIDR() bool {
```

Comment:
Looks like this can be simplified. No need for the switch.

```
func (b *BootstrapConfig) IsValidCIDR() bool {
	clusterCIDRs := strings.Split(b.ClusterCIDR, ",")

	if len(clusterCIDRs) != 1 && len(clusterCIDRs) != 2 {
		return false
	}

	for _, cidr := range clusterCIDRs {
		_, _, err := net.ParseCIDR(cidr)
		if err != nil {
			return false
		}
	}

	return true
}